RadixWallet/RadixConnect/RadixConnect/Mobile/RadixConnectMobileSessionStorage.swift

```diff
@@ -0,0 +1,14 @@
+import SargonUniFFI
+
+// MARK: - SecureSessionStorage
+final class SecureSessionStorage: SessionStorage {
+	@Dependency(\.secureStorageClient) var secureStorageClient
+
+	func saveSession(sessionId: SessionId, encodedSession: Data) async throws {
+		try secureStorageClient.saveRadixConnectMobileSession(sessionId, encodedSession)
+	}
+
+	func loadSession(sessionId: SessionId) async throws -> Data? {
+		try secureStorageClient.loadRadixConnectMobileSession(sessionId)
+	}
+}
```

Comment:
perhaps a small unit tests for this? Just calling `save` and then `load` and assert the correct session data is loaded?